frontend/src/main/pages/ProfilePage.js

```diff
@@ -1,20 +1,62 @@
-import React from "react";
-import { Row, Col } from "react-bootstrap";
+import React, { useState, useEffect } from "react";
+import { Row, Col, Modal, Button, Form } from "react-bootstrap";
 import RoleBadge from "main/components/Profile/RoleBadge";
 import { useCurrentUser } from "main/utils/currentUser";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import {  useBackendMutation } from "main/utils/useBackend";
 
 const ProfilePage = () => {
-
     const { data: currentUser } = useCurrentUser();
+    // Stryker disable all
+    const [showModal, setShowModal] = useState(false);
+    const [phone, setPhone] = useState(() => currentUser?.root?.user?.cellphone);
+    const [newPhone, setNewPhone] = useState(() => currentUser?.root?.user?.cellphone);
+    useEffect(() => {
+        setPhone(currentUser?.root?.user?.cellphone);
```

Comment:
Avoid making new state based on existing state and trying to sync them. You will end up with out of sync situations where `currentUser?...phone` is different from `phone`. Instead you can just read from it directly during render.

```suggestion
    const phone = currentUser?.root?.user?.cellphone;
    const [newPhone, setNewPhone] = useState(() => currentUser?.root?.user?.cellphone);
    useEffect(() => {
```